lib/curl_ntlm_core.h

```diff
@@ -53,6 +53,10 @@
 #define USE_NTLM2SESSION 1
 #endif
 
+#if !defined(USE_NTLM2SESSION)
+#define USE_NTLM2SESSION 0
```

Comment:
I would prefer that we rather make sure to depend on features with `#ifdef` so that we `#undef` features we don't have and we `#define` those we have. I think that's the pattern used more widely already. Defining to zero when we don't have it is an anomaly.